backend/src/data/migrations/20220812131517_add_course_to_vendor_associations.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from 'knex';
+
+enum TableName {
+  COURSES = 'courses',
+  VENDORS = 'vendors',
+  COURSES_TO_VENDORS = 'course_to_vendor',
+}
+
+enum ColumnName {
+  ID = 'id',
+  VENDOR_ID = 'vendor_id',
+  COURSE_ID = 'course_id',
+}
+
+async function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TableName.COURSES_TO_VENDORS, (table) => {
```

Comment:
@fshabanov You're right, one to many, and we don't need additional table `course_to_vendor` for this relation

Reply:
agreed